tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature

```diff
@@ -165,3 +165,50 @@ Feature: Share a file or folder that is inside a space
     When user "Alice" expires the last share
     Then the HTTP status code should be "200"
     And as "Brian" folder "Shares/folder" should not exist
+
+  @issue-enterprise-6423 @env-config
+  Scenario Outline: user cannot share items in the project space with share permission if resharing is denied
+    Given the config "OCIS_ENABLE_RESHARING" has been set to "false"
+    And user "Alice" has shared a space "share sub-item" with settings:
```

Comment:
let's use noun for space names
```suggestion
    And user "Alice" has shared a space "project 101" with settings:
```

Reply:
I haven't space `project 101`. In the Background section Alice creates `share sub-item` space which I use

Reply:
we can do it later. 👍